.gitignore

```diff
@@ -312,6 +312,7 @@ static
 /.terraform.*/
 /.terraform.lock.hcl
 /terraform/variables.tf
+.terraform
```

Comment:
can/should we also remove the terraform directory from this repo?

Reply:
I'm not sure, are the batch jobs moved over? I'm not sure if anything still relies on that folder

Reply:
Hm, I think the only terraform code that's still (somewhat) in use from that directory is the s3 buckets. We should import those into our infra repo when we get a chance.